clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -0,0 +1,574 @@
+        ):
+            raise RuntimeError(
+                f"Cannot cp file of the fileset: {src_identifier} which only mounts to a single file."
+            )
+        dst_context: FilesetContext = self._get_fileset_context(path2)
+
+        src_context.fs.cp_file(
+            self._strip_storage_protocol(
+                src_context.storage_type, src_context.actual_path
+            ),
+            self._strip_storage_protocol(
+                dst_context.storage_type, dst_context.actual_path
+            ),
+        )
+
+    # pylint: disable=W0221
+    def mv(self, path1, path2, **kwargs):
+        """Move a file to another directory.
+         This can move a file to another existing directory.
+         If the target path directory does not exist, an exception will be thrown.
+        :param path1: Virtual src fileset path
+        :param path2: Virtual dst fileset path, should be consistent with the src path fileset identifier
+        :param kwargs: Extra args
+        """
+        path1 = self._pre_process_path(path1)
+        path2 = self._pre_process_path(path2)
+        src_identifier: NameIdentifier = self._extract_identifier(path1)
+        dst_identifier: NameIdentifier = self._extract_identifier(path2)
+        if src_identifier != dst_identifier:
+            raise RuntimeError(
+                f"Destination file path identifier: `{dst_identifier}`"
+                f" should be same with src file path identifier: `{src_identifier}`."
+            )
+        src_context: FilesetContext = self._get_fileset_context(path1)
+        if self._check_mount_single_file(
+            src_context.fileset, src_context.fs, src_context.storage_type
+        ):
+            raise RuntimeError(
+                f"Cannot cp file of the fileset: {src_identifier} which only mounts to a single file."
+            )
+        dst_context: FilesetContext = self._get_fileset_context(path2)
+        src_context.fs.move(
+            self._strip_storage_protocol(
+                src_context.storage_type, src_context.actual_path
+            ),
+            self._strip_storage_protocol(
+                dst_context.storage_type, dst_context.actual_path
+            ),
+        )
+
+    def _rm(self, path):
+        raise RuntimeError("Deprecated method, use `rm_file` method instead.")
+
+    def rm(self, path, recursive=False, maxdepth=None):
+        """Remove a file or directory.
+        :param path: Virtual fileset path
+        :param recursive: Whether to remove the directory recursively.
+                When removing a directory, this parameter should be True.
+        :param maxdepth: The maximum depth to remove the directory recursively.
+        """
+        context: FilesetContext = self._get_fileset_context(path)
+        context.fs.rm(
+            self._strip_storage_protocol(context.storage_type, context.actual_path),
+            recursive,
+            maxdepth,
+        )
```

Comment:
Does this mean that underlying fs should support recursively delete, right?

Reply:
Yeah